src/main/java/gregtech/integration/forestry/recipes/ElectrodeRecipes.java

```diff
@@ -0,0 +1,322 @@
+            addFabricatorRecipe(EnumElectronTube.IRON,
+                    "SXS", "#X#", "XXX",
+                    'S', new UnificationEntry(screw, Iron).toString(),
+                    '#', new UnificationEntry(wireGtSingle, RedAlloy).toString(),
+                    'X', new UnificationEntry(plate, Iron).toString());
+        }
+
+        addFabricatorRecipe(EnumElectronTube.GOLD,
+                "SXS", "#X#", "XXX",
+                'S', new UnificationEntry(screw, Gold).toString(),
+                '#', new UnificationEntry(wireGtSingle, RedAlloy).toString(),
+                'X', new UnificationEntry(plate, Gold).toString());
+
+        addFabricatorRecipe(EnumElectronTube.DIAMOND,
+                "SXS", "#X#", "XXX",
+                'S', new UnificationEntry(screw, Diamond).toString(),
+                '#', new UnificationEntry(wireGtSingle, RedAlloy).toString(),
+                'X', new UnificationEntry(plate, Gold).toString());
+
+        addFabricatorRecipe(EnumElectronTube.OBSIDIAN,
+                " X ", "#X#", "XXX",
+                '#', new UnificationEntry(wireGtSingle, RedAlloy).toString(),
+                'X', new UnificationEntry(plate, Obsidian).toString());
+
+        addFabricatorRecipe(EnumElectronTube.BLAZE,
+                " X ", "#X#", "XXX",
+                '#', new UnificationEntry(wireGtSingle, RedAlloy).toString(),
+                'X', new ItemStack(Items.BLAZE_POWDER));
+
+        addFabricatorRecipe(EnumElectronTube.EMERALD,
+                "SXS", "#X#", "XXX",
+                'S', new UnificationEntry(screw, Emerald).toString(),
+                '#', new UnificationEntry(wireGtSingle, RedAlloy).toString(),
+                'X', new UnificationEntry(plate, Emerald).toString());
+
+        addFabricatorRecipe(EnumElectronTube.APATITE,
+                "SXS", "#X#", "XXX",
+                'S', new UnificationEntry(screw, Apatite).toString(),
+                '#', new UnificationEntry(wireGtSingle, RedAlloy).toString(),
+                'X', new UnificationEntry(plate, Apatite).toString());
+
+        addFabricatorRecipe(EnumElectronTube.LAPIS,
+                "SXS", "#X#", "XXX",
+                'S', new UnificationEntry(screw, Lapis).toString(),
+                '#', new UnificationEntry(wireGtSingle, RedAlloy).toString(),
+                'X', new UnificationEntry(plate, Lapis).toString());
+
+        addFabricatorRecipe(EnumElectronTube.ENDER,
+                " X ", "#X#", "XXX",
+                '#', new UnificationEntry(plate, EnderEye).toString(),
+                'X', new ItemStack(Blocks.END_STONE));
+
+        if (Loader.isModLoaded(GTValues.MODID_XU2)) {
+            addFabricatorRecipe(EnumElectronTube.ORCHID,
+                    " X ", "#X#", "XXX",
+                    '#', new ItemStack(Items.REPEATER),
+                    'X', new ItemStack(Blocks.REDSTONE_ORE));
+        }
+
+        // todo rubber
```

Comment:
Rubber is strange because forestry only adds it if IC2, Tech Reborn, and/or binniecore are loaded. Once we move to mixin, I want to add GT into that list, and then I'll flesh this stuff out